src/bgw/scheduler.c

```diff
@@ -529,10 +529,15 @@ ts_populate_scheduled_job_tuple(ScheduledBgwJob *sjob, Datum *values)
 #endif
 
 static int
+#if PG13_LT
 cmp_next_start(const void *left, const void *right)
 {
 	const ListCell *left_cell = *((ListCell **) left);
 	const ListCell *right_cell = *((ListCell **) right);
+#else
+cmp_next_start(const ListCell *left_cell, const ListCell *right_cell)
+{
+#endif
```

Comment:
A suggestion to do it without directives:
```suggestion
static int
cmp_next_start_void(const void *left, const void *right)
{
	const ListCell *left_cell = *((ListCell **) left);
	const ListCell *right_cell = *((ListCell **) right);
	cmp_next_start(left_cell, right_cell);
}

static int
cmp_next_start(const ListCell *left_cell, const ListCell *right_cell)
{
```
`cmp_next_start` or its declaration should precede `cmp_next_start_void`. (the name is just suggestion)